### PR TITLE
feat(steadybit): add spec

### PR DESCRIPTION
**Pull request by bripkens**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

New autocompletion spec for the [steadybit CLI](https://github.com/steadybit/cli)

**What is the current behavior? (You can also link to an open issue here)**

No autocompletion :trollface: 

**What is the new behavior (if this is a feature change)?**

Auto completion! ❤️ 

**Additional info:**

### CODE REVIEW

File: src/steadybit.ts [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+};
+
+const defRepoCommand: Fig.Subcommand = {
+  name: "def-repo",
+  description:
+    "Change versions and verify a task/policy definition repository state",
+  subcommands: [
+    {
+      name: "set-version",
+      description: "Set the versions in policu and task references",
```
Comment (fedeci) [inline src/steadybit.ts:52]:
```suggestion
      description: "Set the versions in policies and task references",
```

**Review by fedeci — CHANGES_REQUESTED** [review]
Thanks! There is just a small typo to fix.

File: src/steadybit.ts (hunk 1 above)
Comment (bripkens) [reply src/steadybit.ts]: whoops, good catch. Thank you @fedeci!

**Comment by mschrage** [comment]
Also @bripkens it looks like `steadybit` uses commander.js to build the CLI. You might be able to generate this spec automatically using our [integration with commander.js](https://fig.io/docs/guides/autocomplete-for-teams/commander).

**Review by fedeci — APPROVED** [review]

**Pull request merged**